src/dwarf/Gparser.c

```diff
@@ -390,9 +390,9 @@ run_cfi_program (struct dwarf_cursor *c, dwarf_state_record_t *sr,
           if (((ret = read_regnum (as, a, addr, &regnum, arg)) < 0)
               || ((ret = dwarf_read_uleb128 (as, a, addr, &val, arg)) < 0))
             break;
-          set_reg (sr, regnum, DWARF_WHERE_CFAREL, -(val * dci->data_align));
+          set_reg (sr, regnum, DWARF_WHERE_CFAREL, (unw_word_t)(-(unw_sword_t)val));
           Debug (15, "CFA_GNU_negative_offset_extended cfa+0x%lx\n",
-                 (long) -(val * dci->data_align));
+                 (long)(unw_word_t)(-(unw_sword_t)val));
```

Comment:
Today the CI is fixed. Looks like Gparser on win-arm64 needs casts in a few more places:  https://app.travis-ci.com/github/libunwind/libunwind/jobs/571010144
```
C:\Users\travis\build\libunwind\libunwind\src\dwarf\Gparser.c(821): error C2146: syntax error: missing ';' before identifier '__asm__' [C:\Users\travis\build\libunwind\libunwind\bin\windows-x64\aarch64-linux-gnu\src\libunwind.vcxproj]

C:\Users\travis\build\libunwind\libunwind\src\dwarf\Gparser.c(821): warning C4013: '__asm__' undefined; assuming extern returning int [C:\Users\travis\build\libunwind\libunwind\bin\windows-x64\aarch64-linux-gnu\src\libunwind.vcxproj]

C:\Users\travis\build\libunwind\libunwind\src\dwarf\Gparser.c(821): warning C4244: '=': conversion from 'unw_word_t' to 'int', possible loss of data [C:\Users\travis\build\libunwind\libunwind\bin\windows-x64\aarch64-linux-gnu\src\libunwind.vcxproj]

C:\Users\travis\build\libunwind\libunwind\src\dwarf\Gparser.c(821): error C2106: '=': left operand must be l-value [C:\Users\travis\build\libunwind\libunwind\bin\windows-x64\aarch64-linux-gnu\src\libunwind.vcxproj]

C:\Users\travis\build\libunwind\libunwind\src\dwarf\Gparser.c(822): error C2143: syntax error: missing ')' before ':' [C:\Users\travis\build\libunwind\libunwind\bin\windows-x64\aarch64-linux-gnu\src\libunwind.vcxproj]
```